src/common/playlists/PlayList.jsx

```diff
@@ -1,18 +1,24 @@
 import { useSearchFilter } from "common/search/hooks/useSearchFilter";
 import PlayThumbnail from "./PlayThumbnail";
-
+import { useLocation, useNavigate } from "react-router-dom";
 import './playlist.css';
 
 const PlayList = () => {
   const plays = useSearchFilter();
-  return (
-    <>
-      <ol className="list-plays">
-        {plays.map((play, index) => (
-          <PlayThumbnail key={play.id} play={play} />
-        ))}
-      </ol>
-    </>
-  )};
+  const navigate=useNavigate();
+  if(plays===null || plays===undefined){
+    navigate('/play-not-found', {replace: true});
+  }
+  else{
+      return (
+        <>
+          <ol className="list-plays">
+            {plays.map((play, index) => (
```

Comment:
You need to have a conditional rendering of the component here. Check if plays is empty and then render the PlayNotFound component. Do not navigate.

Reply:
i'll try sir to work on this issue